server/remote_cache/action_cache_server/action_cache_server.go

```diff
@@ -173,6 +174,7 @@ func (s *ActionCacheServer) GetActionResult(ctx context.Context, req *repb.GetAc
 	if err := ValidateActionResult(ctx, s.cache, req.GetInstanceName(), req.GetDigestFunction(), rsp); err != nil {
 		return nil, status.NotFoundErrorf("ActionResult (%s) not found: %s", d, err)
 	}
+	s.maybeInlineOutputFiles(ctx, req, rsp, 4*1024*1024)
```

Comment:
Is the maximum message size configurable or do we have a constant for this somewhere?

I made it a parameter since this number will need to be modified when we use this for `ExecuteRequest` (since there are more fields than just `ActionResult` in `ExecuteResponse`).

Reply:
We configure a max recv message size in grpc_server.go, but not a max send msg size. I'm not sure if there is a default limit imposed by the grpc lib that we should reference, or if there are other effective limits we'd need to consider.